webviews/welcome-page/src/components/shortcut/useShortcuts.ts

```diff
@@ -0,0 +1,106 @@
+import { IvyIcons } from '@axonivy/ui-icons';
+import { useTranslation } from 'react-i18next';
+import { useVscode } from '../../util/useVscode';
+import type { Shortcut } from './Shortcut';
+
+export const useShortcuts = () => {
```

Comment:
IMO, it is not needed to have this available shortcuts and as array, or do we expect that they become dynamic in some way?
I would just hard code them into the jsx...